src/pages/product-list-page/product-list-page.js

```diff
@@ -122,9 +122,16 @@ const ProductListPage = ({ category, model }) => {
   }
 
   const categoryText = category.name[language].value.toUpperCase();
-  const itemsToShow = products.map((product, index) => (
-    <ProductListItem key={index} product={product} category={categoryText} />
-  ));
+  const itemsToShow =
+    products.length > 0
```

Comment:
you can use `products.length ?` condition to check if there are items

Reply:
I don't think that we need this change at all. Because there is no difference in the current situation between rendering `null` and empty array. Please, revert it if you can

Reply:
reverted